app/js/components/searchHistory/searchHistoryComponent.jsx

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+
+import './searchHistory.css';
+
+export default class SearchHistoryComponent extends Component{
+    componentDidMount(){}
+    render(){
+        return (
+            <div className="search-history">
+                <h2>Search History</h2>
+                <div className="history-window">
+                    <span>No Search History</span>
+                </div>
+                <div className="select-display">
+                    <label htmlFor="">Display which cohort:
+                    <select name="" id="">
+                        <option value="">Results of last search</option>
+                    </select>
+                </label>
+                </div>
+                <div className="result-window">
+                    <span>No Results</span>
```

Comment:
I assume these are just place holders which we are going to replace with data from the server. Correct?